### PR TITLE
add blobfuse flexvolume driver for kubernetes

### PR DESCRIPTION
**Pull request by andyzhangx**
This is the first version of blobfuse flexvolume driver for kubernetes. User could only deploy blobfuse driver manually by this version, I will create a kubernetes daemonset deployment in the next version.

### CODE REVIEW

File: kubernetes/README.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+cd /etc/kubernetes/volumeplugins/azure~blobfuse/bin
+sudo wget -O blobfuse https://raw.githubusercontent.com/andyzhangx/Demo/master/linux/flexvolume/blobfuse/binary/ubuntu1604-4.4.0-104-generic/blobfuse
+sudo chmod a+x blobfuse
+
+cd /etc/kubernetes/volumeplugins/azure~blobfuse
+sudo wget -O blobfuse https://raw.githubusercontent.com/andyzhangx/Demo/master/linux/flexvolume/blobfuse/blobfuse
+sudo chmod a+x blobfuse
+```
+#### Note:
+Make sure `jq` package is installed on every node.
```
Comment (khenidak) [inline kubernetes/README.md]: nit: we already install jq as per step 1
nit 2: We need a note to tell the user that this is only for debian distros

File: kubernetes/blobfuse [hunk 2]
```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+
+# Notes:
+#  - Please install "jq" package before using this driver.
```
Comment (khenidak) [inline kubernetes/blobfuse]: same nit

**Comment by khenidak** [comment]
LGTM. 

Just one request: Any chance we can add this by default to acs-engine? The installer can run as part of the bootstrap process.

**Comment by andyzhangx** [comment]
close this PR, since the driver and doc would be https://github.com/andyzhangx/kubernetes-drivers/tree/master/flexvolume/blobfuse/ now.

**Pull request closed without merging**